deploy/alibaba-cloud/README.md

```diff
@@ -48,7 +48,7 @@ This repository contains information about how to deploy NVIDIA Triton Inference
 
 # Demo Instruction
 ## Prepare a model repo directory in OSS
-Download the tensorflow inception model via [fetch_model.sh](https://github.com/triton-inference-server/server/blob/main/docs/examples/fetch_models.sh). Then using [ossutil](https://help.aliyun.com/document_detail/50452.html?spm=a2c4g.11186623.6.833.26d66d51dPEytI) , which is a command line tool to use OSS, to upload the model to a certain OSS dir as you want.
+Download the tensorflow inception model via [fetch_model.sh](https://github.com/triton-inference-server/server/blob/r22.10/docs/examples/fetch_models.sh). Then using [ossutil](https://help.aliyun.com/document_detail/50452.html?spm=a2c4g.11186623.6.833.26d66d51dPEytI) , which is a command line tool to use OSS, to upload the model to a certain OSS dir as you want.
```

Comment:
Do we have to change 'main' to 'r22.10'?

Reply:
Hi @tanmayv25 , thank you for you review, I've waived that commit. 
Could you please review it again